x-pack/plugins/transform/public/app/common/pivot_aggs.test.ts

```diff
@@ -67,3 +67,9 @@ describe('getAggConfigFromEsAgg', () => {
     });
   });
 });
+
+describe('isSpecialSortField', () => {
+  test('detects special sort field', () => {
+    expect(isSpecialSortField('_score')).toBe(true);
```

Comment:
nit: would be good to add one or more inputs here to assert they will return false

Reply:
Done in https://github.com/elastic/kibana/pull/101152/commits/a2ae4895d0a9c4f05e028c58299f86520bec0f67